terraform/modules/collection_resources/bucket.tf

```diff
@@ -0,0 +1,120 @@
+#tfsec:ignore:aws-s3-encryption-customer-key: requires public access
+#tfsec:ignore:aws-s3-enable-bucket-encryption: requires public access
+resource "aws_s3_bucket" "collection_resources" {
+  #checkov:skip=CKV2_AWS_6: Public access block is intentionally disabled for this bucket
+  #checkov:skip=CKV2_AWS_62: no need for event notifications
+  #checkov:skip=CKV_AWS_144: cross region replication is overkill when this is only for data transfer
+  #checkov:skip=CKV_AWS_145: requires public access
+  bucket = "${var.prefix}-collection-resources"
+}
+
+resource "aws_s3_bucket_logging" "access_logging" {
+  bucket        = aws_s3_bucket.collection_resources.id
+  target_bucket = aws_s3_bucket.collection_resources_access_logs.id
+  target_prefix = ""
+}
+
+#tfsec:ignore:aws-s3-block-public-acls: Public ACLs are allowed for this bucket
+#tfsec:ignore:aws-s3-block-public-policy: Public policies are allowed for this bucket
+#tfsec:ignore:aws-s3-ignore-public-acls: Public ACLs are allowed for this bucket
+#tfsec:ignore:aws-s3-no-public-buckets: This bucket is intentionally public
+resource "aws_s3_bucket_public_access_block" "collection_resources" {
+  bucket = aws_s3_bucket.collection_resources.id
+
+  #checkov:skip=CKV_AWS_53: Public ACLs are intentionally allowed for this bucket
+  #checkov:skip=CKV_AWS_54: Public policies are intentionally allowed for this bucket
+  #checkov:skip=CKV_AWS_55: Public ACLs are intentionally allowed for this bucket
+  #checkov:skip=CKV_AWS_56: This bucket is intentionally public
+  block_public_acls       = false
+  block_public_policy     = false
+  ignore_public_acls      = false
+  restrict_public_buckets = false
+}
+
+resource "aws_s3_bucket_policy" "public_read" {
+  bucket = aws_s3_bucket.collection_resources.id
+
+  #checkov:skip=CKV_AWS_70: Public access block is intentionally disabled for this bucket
+  policy = jsonencode({
```

Comment:
Could this come out into a policy document data resource, rather than json encoding an object? (and similar elsewhere) 
We're a bit inconsistent across this codebase, but generally aiming towards that method.

(I know this isn't new since my last review, but would be a good bit of tidying)